src/tox_travis/hooks.py

```diff
@@ -11,13 +11,12 @@ def tox_addoption(parser):
         '--travis-after', dest='travis_after', action='store_true',
         help='Exit successfully after all Travis jobs complete successfully.')
 
-    default_toxenv()
-
 
 @tox.hookimpl
 def tox_configure(config):
     """Check for the presence of the added options."""
     if config.option.travis_after:
-        travis_after_monkeypatch()
+        travis_after_monkeypatch(config)
 
     override_ignore_outcome(config)
+    default_toxenv(config)
```

Comment:
> My current thought is that they should remain together since they are essentially the main entry point into the plugin.

Yeah, I think that makes sense. Practially speaking, this is the module that's being imported by Tox, because it's specified in the `setup.py`.